_posts/for-developers/apps/inrupt-tutorial/2020-11-06-00_getting-started.md

```diff
@@ -437,7 +437,9 @@ Node.js installation.
         If there are warnings with regards to the various source maps, you
         can safely ignore them.
 
-7. Open `localhost:1234` in a browser.
+7. Open `localhost:1234` in a browser. If you want, you can also check out the running app on CodeSandobox. Note that you'll have to open the app in a new window (button in the upper right hand corner) to be able to log in.<span> 
```

Comment:
Typo here (GitHub won't allow me suggest the update, i.e. no 'Suggeston' button !), but 'CodeSandobox' should be 'CodeSandbox'.
(I'd also elaborate on the need to open a new Window for it to actually work (i.e. explain why that's needed, as it seems to kinda defeat the purpose of CodeSandbox - i.e. it should be all runnable right there for you), and maybe include a screenshot of the 'button in the upper right-hand corner', or at least allude to it being the button with the 'Open In New Window' tooltip)): 
![image](https://user-images.githubusercontent.com/961445/98551141-e35d9f80-2294-11eb-8957-be86b04abe94.png)

Reply:
Typo has been fixed. Any change you can do a PR with the additional adjusted language?